packages/client/components/PokerCardDeck.tsx

```diff
@@ -182,11 +184,11 @@ const PokerCardDeck = (props: Props) => {
     }
   }, [maxSlide, isCollapsed])
   // const transform = maxSlide > 0 && !isCollapsed ? `translateX(${swipe.translateX}px)` : undefined
-  if (isSpectating) return null
   return (
     <Deck
       ref={deckRef}
       left={left}
+      isSpectating={!!isSpectating}
```

Comment:
+1 I'd set this to a boolean on L52. the earlier we can establish strict types, the better!